stable/harbor-copy-secret/README.md

```diff
@@ -0,0 +1,17 @@
+# harbor-copy-secret
+
+A helm chart that copies secret content between namespaces, can be useful when a secret is created in a
+workspace namespace and needs to be copied to a release namespace.
+
+## How it works
+
+Chart can be activated by enabling the `enabled` flag, it will then copy the secret content from the source namespace to the target namespace.
+
+```yaml
+harborCopySecret:
+  enabled: true
```

Comment:
Why would you deploy this chart not enabled? Is this necessary?

Reply:
Discussed via slack, this is necessary for conditionally activating this helm chart installation via values.